backend/functions/lib/adapters/gateways/mock-pacer-token-secret.gateway.ts

```diff
@@ -20,4 +20,12 @@ export class MockPacerTokenSecretGateway implements PacerTokenSecretInterface {
     this._token = token;
     return Promise.resolve(undefined);
   }
+
+  getPacerPasswordFromSecrets(): Promise<string> {
+    return Promise.resolve('');
```

Comment:
A little odd that we are defining a get method that returns nothing.  We may not need it to for now, but it's odd.